unlock-app/src/errors.js

```diff
@@ -12,3 +12,9 @@ export const FORM_EXPIRATION_DURATION_INVALID =
   'FORM_EXPIRATION_DURATION_INVALID'
 export const FORM_MAX_KEYS_INVALID = 'FORM_MAX_KEYS_INVALID'
 export const FORM_KEY_PRICE_INVALID = 'FORM_KEY_PRICE_INVALID'
```

Comment:
Does this refer to the error that was removed in line 36 of `ErrorMessage.js`?

Reply:
Oups! Fixed! PTAL ;)